blockchain/tx_pool.go

```diff
@@ -445,6 +447,11 @@ func (pool *TxPool) reset(oldHead, newHead *types.Header) {
 	// Check the queue and move transactions over to the pending if possible
 	// or remove those that have become invalid
 	pool.promoteExecutables(nil)
+
+	// Update all fork indicator by next pending block number.
+	next := new(big.Int).Add(newHead.Number, big.NewInt(1))
+	// TODO-Klaytn-AccessList: Make another hardfork for eip2718 instead of London
```

Comment:
Can you explain what this comment means?

Reply:
Because it already applied london hardfork at Baobob network without this transaction, it needs to additonal hard fork to include this changes.